.github/workflows/submodule_update.yml

```diff
@@ -4,50 +4,94 @@ on:
   push:
     branches:
       - 'master'
-      - '2.10'
+      - '[2-9].\d+'
+
+env:
+  EE_FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
 
 jobs:
   tarantool-ee:
```

Comment:
Not a big deal, but doesn't it look more consistent if it would be `update-tarantool-ee` or `update-ee`?

Reply:
I agree, it's better this way. Now the jobs and task id's are named `update-ee` and `test-sdk`.